packages/x-charts/src/ChartsGrid/gridClasses.ts

```diff
@@ -0,0 +1,25 @@
+import generateUtilityClass from '@mui/utils/generateUtilityClass';
+import generateUtilityClasses from '@mui/utils/generateUtilityClasses';
+
+export interface ChartsGridClasses {
+  /** Styles applied to the root element. */
+  root: string;
+  /** Styles applied to the main line element. */
+  line: string;
+  /** Styles applied to x-axes. */
+  horizontalLine: string;
+  /** Styles applied to y-axes. */
+  verticalLine: string;
+}
+
+export type ChartsGridClassKey = keyof ChartsGridClasses;
+
+export function getGridUtilityClass(slot: string) {
```

Comment:
Nitpick: I'd suggest considering prefixing `Grid` with `Charts` everywhere, because of how common the `grid` word is. 🙈 🤔 
```suggestion
export function getChartsGridUtilityClass(slot: string) {
```